src/folders_and_files/metadata.md

```diff
@@ -147,6 +147,145 @@ https://github.com/jeroen/jsonlite
     writeLines(data, file="myData.json")
 ```
 
+## Interoperability issues
```

Comment:
@VisLab I am tagging you here because this kind of issue could be more annoying for annotation and HED than for other aspects of BIDS. 
This is not urgent and even if I merge this "too soon", please feel free to open issues and amend with PR laters.